Examples/PointOfSale/PointOfSale.Handlers.CustomerHandlers/Program.cs

```diff
@@ -21,7 +22,7 @@ public class BusStateChangedHandler : IBusStateChangedHandler
             public void OnBusStarted(IBusConfiguration config, IBus bus)
             {
 #if DEBUG
-                if (config.IsConsoleMode)
+                if ((config as IWindowsBusConfiguration).IsConsoleMode)
```

Comment:
Use an explicit cast, if the cast doesn't work you'll get an `InvalidCastException` opposed to a `NullReferenceException` you'd get when using `as`.